src/core/server/opensearch/opensearch_config.test.ts

```diff
@@ -437,12 +437,12 @@ describe('deprecations', () => {
     `);
   });
 
-  it('logs a warning if elasticsearch.requestHeadersWhitelist is set and opensearch.requestHeadersWhitelist is not', () => {
-    const { messages } = applyLegacyDeprecations({ requestHeadersWhitelist: [''] });
+  it('logs a warning if elasticsearch.requestHeadersAllowlist is set and opensearch.requestHeadersAllowlist is not', () => {
+    const { messages } = applyLegacyDeprecations({ requestHeadersAllowlist: [''] });
     expect(messages).toMatchInlineSnapshot(`
       Array [
-        "\\"elasticsearch.requestHeadersWhitelist\\" is deprecated and has been replaced by \\"opensearch.requestHeadersWhitelist\\"",
-        "\\"opensearch.requestHeadersWhitelist\\" is deprecated and has been replaced by \\"opensearch.requestHeadersAllowlist\\"",
+        "\\"elasticsearch.requestHeadersAllowlist\\" is deprecated and has been replaced by \\"opensearch.requestHeadersAllowlist\\"",
```

Comment:
i think since it's 3.0 we can remove the `elasticsearch.` like we do not allow providers to set `elasticsearch.requestHeadersAllowlist` then can only set `opensearch.requestHeadersAllowlist`

Reply:
Oops this was a mistake, I will fix in next commit